OracleUnifiedDirectorySM/imagetool/12.2.1.4.0/README.md

```diff
@@ -1,289 +1,18 @@
 Building an Oracle Unified Directory Services Manager Image with Weblogic Image Tool
 ====================================================================================
 
-## Contents
+# Introduction
 
-1. [Introduction](#1-introduction)
-2. [Prerequisites](#2-prerequisites)
-3. [Setup WebLogic Image Tool](#3-setup-weblogic-image-tool)
-4. [Download the required packages/installers](#4-download-the-required-packagesinstallers)
-5. [Additional build files](#5-additional-build-files)
-6. [Additional build commands](#6-additional-build-commands)
-7. [Steps to Create Oracle Unified Directory Services Manager image](#7-steps-to-create-oracle-unified-directory-services-manager-image)
-8. [Generate Sample dockerfile with imagetool](#8-generate-sample-dockerfile-with-imagetool)
+For instructions on building an OUDSM image with WebLogic Image Tool see [Create or update image](https://oracle.github.io/fmw-kubernetes/oudsm/create-or-update-image#create-or-update-an-oracle-unified-directory-services-manager-image-using-the-weblogic-image-tool).
```

Comment:
```suggestion
Detailed instructions on how to [create or update an Oracle Unified Directory Services Manager container image](https://oracle.github.io/fmw-kubernetes/oudsm/create-or-update-image#create-or-update-an-oracle-unified-directory-services-manager-image-using-the-weblogic-image-tool) using the WebLogic Image Tool can be found in the [Oracle Fusion Middleware on Kubernetes](https://oracle.github.io/fmw-kubernetes/) documentation.
```

However, the provided URL returns a 404 error.

Reply:
The URL link will be available once we have it published. It is still undergoing last minute checks.